tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
are we using this declaration?

Reply:
was use to generate .d.ts file for `tokens`. I guess if we're only using `tsc` for tokens or any package, we can include only the specific folder to process, to be clear...